Dockerfile

```diff
@@ -61,6 +61,7 @@ RUN GOARCH="$(xcputranslate translate -targetplatform ${TARGETPLATFORM} -field a
     -X 'main.date=$CREATED' \
     -X 'main.commit=$COMMIT' \
     " -o app cmd/updater/main.go
+RUN mkdir -p /tmp/data

```

Comment:
Very minor detail/optimization, can you move this line right after `FROM --platform=$BUILDPLATFORM base AS build`? That way it isn't executed on every code change/rebuilds (assuming layer caching). Not a big deal though 😉